src/rt-app.c

```diff
@@ -686,6 +670,40 @@ void *thread_body(void *arg)
 			exit(EXIT_FAILURE);
 	}
 
+	data->current_prio = sched_prio;
+}
+
+void *thread_body(void *arg)
+{
+	thread_data_t *data = (thread_data_t*) arg;
+	phase_data_t *pdata;
+	log_data_t ldata;
+	struct sched_param param;
```

Comment:
oops, you are right. I wanted to postponed deadline case but i have done it wrong